components/availability/src/Availability.svelte

```diff
@@ -88,7 +83,6 @@
   export let event_buffer: number;
   export let free_color: string;
   export let mandate_top_of_hour: boolean;
-  export let max_bookable_slots: number;
```

Comment:
This is considered a breaking change. Can you please update `components/availability/CHANGELOG.md` accordingly.

Reply:
This component is unreleased, so we can still make breaking changes without documenting them.

Reply:
Oh yes you're right, forgot that this isn't release yet.